packages/components/src/utils/unit-values.ts

```diff
@@ -0,0 +1,48 @@
+/**
+ * Parses a number and unit from a value.
+ *
+ * @param  toParse Value to parse
+ *
+ * @return  The extracted number and unit.
+ */
+export function parseUnitValue(
+	toParse: string | number
+): [ number | string | undefined, string | undefined ] {
```

Comment:
Why can the returned `value` be a `string` ?

Instead of setting the value of `num` to an empty string, couldn't we set it to `undefined` instead?